### PR TITLE
r-ggplot2 is part of the conda package

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
Testing busco installation for @bedroesb.

File: tools/busco/busco.xml [hunk 1]
```diff
@@ -1,14 +1,11 @@
-<tool id="busco" name="Busco" version="3.0.2">
```
Comment (nsoranzo) [inline tools/busco/busco.xml]: We could use a macro to easily keep tool and requirement version in sync.

File: tools/busco/busco.xml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+<tool id="busco" name="Busco" version="@VERSION@+galaxy0">
     <description>assess genome assembly and annotation completeness</description>
+    <macros>
+        <token name="@VERSION@">3.0.2</token>
+    </macros>
     <requirements>
-        <requirement type="package" version="3.0.2">busco</requirement>
-        <requirement type="package" version="2.2.1">r-ggplot2</requirement>
+        <requirement type="package" version="@VERSION@">busco</requirement>
+        <requirement type="package" version="3.5.1">r-base</requirement>
```
Comment (nsoranzo) [inline tools/busco/busco.xml]: What about using a more recent conda version instead/also?
```patch
diff --git a/.travis.yml b/.travis.yml
index 50a87ac..f422111 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -81,7 +81,7 @@ install:
   - pip install planemo
   - planemo conda_init
   - export PATH="$PLANEMO_CONDA_PREFIX/bin:$PATH"
-  - conda install -y -c conda-forge conda=4.3.34 
+  - conda install -y -c conda-forge conda=4.5.12
   - planemo --version
   - conda --version
   - git diff --quiet "$TRAVIS_COMMIT_RANGE" -- ; GIT_DIFF_EXIT_CODE=$?
```

Comment (bgruening) [reply tools/busco/busco.xml]: Sure, but then let's use 4.6.4. I also rebuild busco on bioconda with explicit r-base, I guess this is the real problem.

Comment (nsoranzo) [reply tools/busco/busco.xml]: Using 4.6.4 actually worked (after a while) without the `r-base` requirement. 4.5.12 was stuck for a long time and I killed it.

Comment (bgruening) [reply tools/busco/busco.xml]: Is this while below 10min :)
I don't think there is any harm to include r-base. You call Nicola!

Comment (nsoranzo) [reply tools/busco/busco.xml]: We can do both?

Comment (nsoranzo) [reply tools/busco/busco.xml]: Another observation: `--strict-channel-priority` option of conda 4.6 doesn't seem to help at all :cry:

**Comment by bgruening** [comment]
Travis is green, github is just not updated.

**Pull request merged**